doc/function_types.md

```diff
@@ -1144,3 +1144,66 @@ including ensmallen:
 This can be useful for situations where you know that the checks should be
 ignored.  However, be aware that the code may fail to compile and give more
 confusing and difficult error messages!
+
+
+## Multi objective functions
+
+A multi objective function is a set of functions or objectives that are
+simultaneously being optimised for. For this, a class/struct with the following
+API must be used:
+
+<details open>
+<summary>Click to collapse/expand example code.
+</summary>
+
+```c++
+class ArbitraryMultiObjectiveFunction
+{
+private:
```

Comment:
Very picky comment, you forgot the space :)

Reply:
On it.